scripts/rust_integration_tests.sh

```diff
@@ -24,7 +24,7 @@ if [ ! -f ${ROOT}/bundle.tar.gz ]; then
 fi
 touch ${LOGFILE}
 
-sudo YOUKI_LOG_LEVEL="error" ${ROOT}/integration_test run --runtime "$RUNTIME" --runtimetest ${ROOT}/runtimetest > $LOGFILE
+sudo ${ROOT}/integration_test run --runtime "$RUNTIME" --runtimetest ${ROOT}/runtimetest > $LOGFILE
```

Comment:
We should be also passing the --log-level flag here, set to error, right?

Reply:
No, this script is calling the integration_test binary, which don't take the `log-level` flag. At least I did not make the change in this PR. The actual code that calls the `runtime` is changed to add the `log-level` flag. The integration_test binary uses `YOUKI_INTEGRATION_LOG_LEVEL` env variable. https://github.com/containers/youki/blob/1a6d1f4bd7553e971d6d787698a9732836188444/tests/rust-integration-tests/integration_test/src/logger.rs#L6

It is possible that the original intent is `YOUKI_INTEGRATION_LOG_LEVEL`?